gulpfile.babel.js

```diff
@@ -110,6 +115,63 @@ const cleanSharedConfig = () => del(config.files.src.derivedConfigs);
 
 export const distributeSharedConfig = gulp.series(cleanSharedConfig, distributeConfig);
 
+const cleanSassLocaleData = () => del([config.files.src.sassLocales]);
+
+const generateSassLocaleData = () => {
```

Comment:
I wonder if it's worth extracting (at least some of) this into a module. There's quite a bit going on here, and that would make it more testable.